.github/workflows/tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
```

Comment:
why removing 3.6?

Reply:
this was just to figure out that the python 3.6 version has problems installing the packages mentioned in the `setup.cfg`. This should be fixed, dont know how. @Luthaf will probably take a look on this soon

Reply:
here this is the error https://github.com/cosmo-epfl/sklearn-cosmo/pull/9/checks?check_run_id=1405315758

Reply:
Once https://github.com/cosmo-epfl/sklearn-cosmo/pull/16 is merged, rebasing this PR should fix this issue!

Reply:
works